src/main.cpp

```diff
@@ -6909,6 +6909,22 @@ void ProcessTxBaseMsg(const CTransactionBase& txBase, CNode* pfrom)
     }
 }
 
+bool static ValidateStream(CNode* pfrom, CDataStream& vRecv, const char* msgTypeName)
```

Comment:
Given that this function is `CInv` specific, I believe it would be clearer in future to name it something like `ValidateInvStream()`, or `ValidateInvStreamSize()`.

Reply:
Done